test/expected/append.out

```diff
@@ -130,19 +129,17 @@ ORDER BY time LIMIT 3;
 psql:include/append.sql:68: NOTICE:  Stable function now_s() called!
 psql:include/append.sql:68: NOTICE:  Stable function now_s() called!
 psql:include/append.sql:68: NOTICE:  Stable function now_s() called!
-psql:include/append.sql:68: NOTICE:  Stable function now_s() called!
 psql:include/append.sql:68: NOTICE:  Stable function now_s() called!
                                               QUERY PLAN                                               
 -------------------------------------------------------------------------------------------------------
  Limit
    ->  Custom Scan (ConstraintAwareAppend)
          Hypertable: append_test
          Chunks left after exclusion: 1
-         ->  Merge Append
-               Sort Key: _hyper_1_3_chunk."time"
+         ->  Append
```

Comment:
The comment for this test (line 124) is no longer accurate, since we're no creating an Append plan instead of a Merge Append

Reply:
we are not creating a special node